### PR TITLE
feat: [PROD-9516] use vertical color indicator for modified inputs

### PR DESCRIPTION
**Pull request by ardvi**

### CODE REVIEW

File: src/inputs/style.js
Comment (csm-thu) [inline src/inputs/style.js:1]: Could we rename this file ? We usually use `style.js` for the style of a single component, but in this case, this style is used for multiple component (but not all components in _src/inputs_ though).

File: src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js [hunk 1]
```diff
@@ -71,7 +73,7 @@ export const BasicSliderInput = (props) => {
           value={getValue(value)}
           sx={sliderStyle}
           color={color}
-          size="small"
+          variant="small"
```
Comment (csm-thu) [inline src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js:76]: Note: this fix should probably be in a different commit, especially if it fixes a bug from a released version (so it can be visible in the release notes) :slightly_smiling_face:

File: src/inputs/style.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    marginLeft: '15px',
+    marginRight: '15px',
+    marginTop: 'auto',
+    marginBottom: 'auto',
+  },
+  loadingSpinner: {
+    marginTop: 'auto',
+    marginBottom: 'auto',
+  },
+}));
```
[the remark this answers is not in the document]
Comment (COSMO-EMI) [reply src/inputs/style.js:42]: I had to put the table part here too since style is generated with the `makeStyles` function. I understand I should only create a dict that contains CSS for every input and use `makeStyles` in each input separately, I just don't know the syntax yet.

File: src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js (hunk 1 above)
Comment (COSMO-EMI) [reply src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js:76]: Agreed. Moreover this is not even a fix.

File: src/inputs/style.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (COSMO-EMI) [reply src/inputs/style.js]: Merci @csm-thu pour le fix!

File: src/inputs/style.js
Comment (COSMO-EMI) [reply src/inputs/style.js:1]: `dirtyStyle.js` ?

Comment (csm-thu) [reply src/inputs/style.js:1]: Not so sure anymore that we must absolutely rename it :sweat_smile: 
I can't find a better name yet, but it won't be a breaking change if we rename it in the future, so let's keep it that way for now :+1:
